#! /bin/sh /usr/share/dpatch/dpatch-run
## package_version.dpatch by Samuel Mimram <smimram@debian.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Define the package_version to avoid necessity for configure.

@DPATCH@
diff -urNad sysprof-1.0.7~/module/sysprof-module.c sysprof-1.0.7/module/sysprof-module.c
--- sysprof-1.0.7~/module/sysprof-module.c	2006-11-20 10:12:04.000000000 +0000
+++ sysprof-1.0.7/module/sysprof-module.c	2006-11-20 10:14:07.000000000 +0000
@@ -19,6 +19,8 @@
  * Foundation, Inc., 59 Temple Place - Suite 330, Boston, MA 02111-1307, USA.
  */
 
+#define PACKAGE_VERSION "1.0.7-debian"
+
 #ifdef CONFIG_SMP
 # define __SMP__
 #endif
@@ -36,8 +38,6 @@
 
 #include "sysprof-module.h"
 
-#include "../config.h"
-
 #include <linux/version.h>
 
 #if LINUX_VERSION_CODE < KERNEL_VERSION(2,6,18)
